pom.xml

```diff
@@ -7,10 +7,11 @@
     <relativePath />
   </parent>
 
-  <groupId>org.jenkins-ci.modules</groupId>
+  <!-- change to generate incremental 0001 -->
+  <groupId>org.jenkins-ci.plugins</groupId> <!-- we use the modules groupId -->
```

Comment:
Revert this. That is why you could not publish incrementals.

Reply:
yep, I know, but we want to change the groupdId, isn't it?

Reply:
The JEP proposes leaving it alone. There is no requirement that a plugin have a specific kind of `groupId`. See `maven-plugin` for example.